src/spec_coerce/core.cljc

```diff
@@ -182,11 +182,33 @@
        (URI. x)
        x)))
 
+(defn type->sym [x]
+  (cond (int? x)     `integer?
+        (float? x)   `float?
+        (boolean? x) `boolean?   ;; pointless but valid
+        ;;(symbol? x)  `symbol?  ;; doesn't work.
+        ;;(ident? x)   `ident?   ;; doesn't work.
+        (string? x)  `string?
+        (keyword? x) `keyword?
+        (uuid? x)    `uuid?
+        (nil? x)     `nil?       ;; even more pointless but stil valid
+
+        ;;#?(:clj (uri? x))     #?(:clj `uri?) ;; doesn't work.
+        #?(:clj (decimal? x)) #?(:clj `decimal?)))
+
+(defn spec-is-homogenous-set? [x]
```

Comment:
```suggestion
(defn spec-is-homogeneous-set? [x]
```

typo

Reply:
Oh, I think both spellings are correct actually, but I'll change it

Reply:
you are probably right, I just sent on google and it suggested the other, thanks

Reply:
Well now I'm curious though, and it looks like the spelling I used is "proscribed," meaning acknowledged but not recommended. I'll keep that in mind from now on.

Reply:
just for fun, I found this: https://www.grammar.com/homogenous_vs._homogeneous

Reply:
Heh, even she spells it wrong while trying to reinforce that they are different.